migrations/Version202106170629531101_taoQtiItem.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiItem\model\search\Filter\HasValueFilter;
+use oat\taoQtiItem\model\search\Filter\NotBase64ContentFilter;
+use oat\taoQtiItem\model\search\Filter\NotJsonFilter;
+use oat\taoQtiItem\model\search\QtiItemContentTokenizer;
+
+final class Version202106170629531101_taoQtiItem extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->getServiceManager()->register(QtiItemContentTokenizer::SERVICE_ID, new QtiItemContentTokenizer(
+            [
+                QtiItemContentTokenizer::OPTION_FILTERS => [
+                    new HasValueFilter(),
+                    new NotJsonFilter(),
+                    new NotBase64ContentFilter(),
+                ]
+            ]
+        ));
```

Comment:
It is not related to any PSR or whatever, but when I read the code like this I am not sure which value is passed to which object. I recommend to indent like this. This way you can see exactly who is receiving what. Wdyt?

```suggestion
        $this->getServiceManager()->register(
            QtiItemContentTokenizer::SERVICE_ID, 
            new QtiItemContentTokenizer(
                [
                    QtiItemContentTokenizer::OPTION_FILTERS => [
                        new HasValueFilter(),
                        new NotJsonFilter(),
                        new NotBase64ContentFilter(),
                    ]
                ]
            )
        );
```